app/models/registration.rb

```diff
@@ -46,16 +46,19 @@ class Registration < ApplicationRecord
 
   serialize :roles, coder: YAML
 
+  # Set a `registered_at` timestamp for newly created records,
+  #   but only if there is no value already specified from the outside
+  after_initialize :mark_registered_at, if: :new_record?, unless: :registered_at?
```

Comment:
Why did you move this here?

Reply:
Same comment as elsewhere - the only thing I can think is the merge with main, but then I don't understand why it would be showing up in the diff

Reply:
Okay, this one is actually the only one that is not a choke: RuboCop wants `before_create` (which was not present in this class before!) to be placed after `after_initialize`.

Why did you choose to move the existing `after_initialize` up, rather than moving the new `before_create` down?

Reply:
That was Rubocop's autocorrect solution - but I've moved the `after_create` down now instead